### PR TITLE
Fix links to etcd public documentation

### PR DESCRIPTION
**Pull request by tengqm**
This PR contains three commits intentionally:

1. Wrap long lines and fix the indentation and other style issues;
2. Fix links to etcd official documentation which is published on its own site;
3. Drop outdated information related to v1.15 or older.

closes: #26542

### CODE REVIEW

**Comment by tengqm** [comment]
Preview link: https://deploy-preview-26564--kubernetes-io-master-staging.netlify.app/docs/tasks/administer-cluster/configure-upgrade-etcd/

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 1]
```diff
@@ -50,33 +55,43 @@ Use a single-node etcd cluster only for testing purpose.
 
 1. Run the following:
 
-    ```sh
-    ./etcd --listen-client-urls=http://$PRIVATE_IP:2379 --advertise-client-urls=http://$PRIVATE_IP:2379
-    ```
+   ```sh
+   etcd --listen-client-urls=http://$PRIVATE_IP:2379 \
```
Comment (kbhawkey) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md:59]: Hi @tengqm .
What are the benefits of reducing the number of characters per line (readability, easier to translate)?

Comment (tengqm) [reply content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md:59]: The benefit is about readability, e.g. number of arguments passed.
Readability is also improved when you are reading the doc on a pad.
It is not related to translation.

**Review by sftim — COMMENTED** [review]
Thanks @tengqm 
/approve

**Comment by sftim** [comment]
The 3 commits look OK to keep separate; I recommend _not_ doing a squash merge.

**Comment by kbhawkey** [comment]
Thanks @tengqm .
/lgtm

**Pull request merged**